dashboards/jupyterhub.libsonnet

```diff
@@ -20,7 +20,7 @@ local prometheus = grafana.prometheus;
    */
   componentLabel(component, cmp='=', namespace='$hub')::
     std.format(
-      'kube_pod_labels{label_app="jupyterhub", label_component%s"%s" %s}',
+      'group(kube_pod_labels{label_app="jupyterhub", label_component%s"%s" %s}) by (label_component,pod,namespace)',
```

Comment:
I think namespace should be conditional here, set only if `namespace` parameter is not set to null, similar to the conditional in line 27.

Reply:
Also split this into multiple lines like the other queries?

Reply:
Let's also add a comment here on why group is used?